modules/ROOT/pages/cassandra/cassandra-xml-maven.adoc

```diff
@@ -0,0 +1,59 @@
+= XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+XML and POM file configuration information and XML-related usage topics.
+
+== Add a Namespace for the Connector
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/cassandra-db
+http://www.mulesoft.org/schema/mule/cassandra-db/current/mule-cassandra-db.xsd
+----
+
+== Configure the Maven POM File
+When creating an app manually from the XML, set up the pom.xml for your project:
+
+ [source,xml,linenums]
+----
+<repositories>
+   <repository>
+        <id>mule-ee-releases</id>
+        <name>MuleEE Releases Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases-ee/</url>
+    <repository>
+        <id>mule-ee-snapshots</id>
+        <name>MuleEE Snapshots Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/ci-snapshots/</url>
+    </repository>
+</repositories>
+----
+
+ This snippet specifies a project dependency for a specific release version:
+
+ [source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.connectors</groupId>
+    <artifactId>mule-cassandra-connector</artifactId>
+    <version>RELEASE</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+ This snippet adds and specifies a project dependency for the latest version:
+
+ [source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-cassandra-connector</artifactId>
+    <version>LATEST</version>
+</dependency>
+----
+
+== See Also
+* https://forums.mulesoft.com[MuleSoft Forum]
```

Comment:
Instead of generic links, can we point people to MuleSoft doc that explains things? What is the relationship of this XML to a Mule app? What am I doing? Why?

Reply:
Agreed, we should decide this for all templates.